charts/aws-ebs-csi-driver/templates/poddisruptionbudget-controller.yaml

```diff
@@ -11,9 +11,16 @@ spec:
     matchLabels:
       app: ebs-csi-controller
       {{- include "aws-ebs-csi-driver.selectorLabels" . | nindent 6 }}
+  {{- if (semverCompare ">= 1.27-0" .Capabilities.KubeVersion.Version) }}
+  {{- with .Values.controller.podDisruptionBudget.unhealthyPodEvictionPolicy }}
+  unhealthyPodEvictionPolicy: {{ . }}
+  {{- end }}
```

Comment:
Can we change lines 15-17 to 

```
unhealthyPodEvictionPolicy: {{ .Values.controller.podDisruptionBudget.unhealthyPodEvictionPolicy }}
```

To match what we do a few lines down in this file and remove the extra `end`?

Reply:
That won't work when using a `with` statement, need to use `.`